src/sphinx.md

```diff
@@ -256,6 +266,9 @@ bytes. For request packets:
 
 ## SURBs
 
+// TODO make it clear where are surb IIRC start of payload and we know if it is here from the action
+// type.
```

Comment:
This is covered in the fragmentation chapter. We know if there are SURBs in the payload from the fragment header; it's not part of the action any more.

Reply:
yes, just could help first reading of the spec.